React/Base/RCTUIKit.h

```diff
@@ -394,8 +394,8 @@ CGPathRef UIBezierPathCreateCGPathRef(UIBezierPath *path);
 
 - (void)setNeedsDisplay;
 
-// An override of an undocumented API that controls the layer's masksToBounds property
-@property (nonatomic) BOOL clipsToBounds;
+// FUTURE: When building with Xcode 15, we can remove this override since it's now declared on NSView
+@property BOOL clipsToBounds;
```

Comment:
Do you mean Xcode 15 as a minimum? I guess we don't really declare minimum Xcode versions, so effectively: "When CI is building with Xcode 15"

Reply:
Good call - attempting to clarify the comment with that more concrete future point.